Mathlib/RingTheory/Etale/Basic.lean

```diff
@@ -26,58 +27,78 @@ localization at an element.
 
 open scoped TensorProduct
 
-universe u
+universe u v
 
 namespace Algebra
 
-section
+variable {R : Type u} {A : Type v} {B : Type*} [CommRing R] [CommRing A] [Algebra R A]
+  [CommRing B] [Algebra R B]
 
-variable (R : Type u) [CommRing R]
-variable (A : Type u) [CommRing A] [Algebra R A]
+section
 
-/-- An `R` algebra `A` is formally étale if for every `R`-algebra, every square-zero ideal
-`I : Ideal B` and `f : A →ₐ[R] B ⧸ I`, there exists exactly one lift `A →ₐ[R] B`. -/
+variable (R A) in
+/-- An `R`-algebra `A` is formally etale if both `Ω[A⁄R]` and `H¹(L_{S/R})` are zero.
```

Comment:
```suggestion
/-- An `R`-algebra `A` is formally etale if both `Ω[A⁄R]` and `H¹(L_{A/R})` are zero.
```
Or is this some notation I don't know about?

Reply:
No I just copy pasted and forgot to change them. Thanks!